config/webpack/plugin/TerraDevSite.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const { DefinePlugin } = require('webpack');
+const SetupPlugin = require('./SetupPlugin');
+const GeneratePlugin = require('./GeneratePlugin');
+const loadSiteConfig = require('../../../scripts/generate-app-config/loadSiteConfig');
+
+/**
+ * Generate a terra-dev-site
+ */
+class TerraDevSite {
+  constructor({ env = {}, sites = [] } = {}) {
+    // Sites to generate. Add the default site then load the config if not present
```

Comment:
Where is the sites configuration documented? Seems like it requires either a siteConfig or configFileName and a default config path. Does it also need an indexPath?

Reply:
Sites is an intentionally undocumented experimental feature.

Reply:
Looks like prefix, basename, filename, entry and indexPath are also expected